src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,37 @@
+<div class="footer">
+  <div class="footer-body">
+    <img src="../../../img/footer.png" class="footer-img" />
+    <div class="information">
```

Comment:
```suggestion
    <address class="information">
```
これも使えますね
https://developer.mozilla.org/ja/docs/Web/HTML/Element/address